oss-internship-2020/guetzli/guetzli_transaction.cc

```diff
@@ -0,0 +1,120 @@
+#include "guetzli_transaction.h"
+
+#include <iostream>
+#include <memory>
+
+namespace guetzli {
+namespace sandbox {
+
+absl::Status GuetzliTransaction::Init() {
```

Comment:
Spliting `Init` and `Main` does not make much sense here
`Init` is not general initialization and is not supposed to be reused for multiple `Main` calls